common/opencv.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+/*
+ * This header is needed, because I'm getting compile errors when trying to include
+ * OpenCV because the KEY_UP and KEY_DOWN macros are already defined (on Ubuntu 18.04).
+ * 	- AD
+ */
```

Comment:
YUCK :/ Any clue where?

Reply:
No :-/. I might install an IDE on my Ubuntu VM and try to track it down though. I don't really like this..

Reply:
From a quick search on my Ubuntu 18 install it looks like /usr/include/linux/input-event-codes.h is a likely candidate :/

Reply:
Google suggests that adding -H to the g++ commandline might list how it gets included